lib/transbank/sdk/webpay/webpay_plus/transaction.rb

```diff
@@ -0,0 +1,105 @@
+module Transbank
+  module Webpay
+    module WebpayPlus
+      class Transaction
+        extend Transbank::Utils::NetHelper
+        CREATE_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+        COMMIT_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+        TRANSACTION_STATUS_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+        REFUND_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions/:token/refunds'
+        class << self
+
+          def create(buy_order:, session_id:, amount:, return_url:, options: nil)
+            if options.nil?
+              api_key = default_integration_params[:api_key]
+              commerce_code = default_integration_params[:commerce_code]
+              base_url = default_integration_params[:base_url]
+            else
+              api_key = options.api_key || default_integration_params[:api_key]
+              commerce_code = options.commerce_code || default_integration_params[:api_key]
+              base_url = WebpayPlus::Base.integration_types[options.integration_type] || default_integration_params[:base_url]
```

Comment:
you can refactor this if using `&` like `api_key = options&.api_key || default_integration_params[:api_key]`